docs/_documentations/codewind-vscode-quick-guide.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: docs
+title: Codewind in VS Code
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+keywords: Codewind, VS Code, microservice
+duration: 5 minutes
+permalink: codewind-vscode-quick-guide
+type: document
+parent: Quick guides 
+order: 1
+---
+
+# Getting Started with Codewind in VS Code
```

Comment:
Same as for the eclipse guide, do we need this for the minikube guide since it's local only?